.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -6,3 +6,6 @@ contact_links:
   - name: Cardano API Issues
     url: https://github.com/IntersectMBO/cardano-api/issues
     about: Report API related issues here
+  - name: Cardano Node Release
```

Comment:
Wouldn't that make a duplicate entry in issues list? One from `node_release.md` and the other from this?

Reply:
I'm not sure. How can we test it?

Reply:
By forking the repo 🙂

Reply:
You were right, @carbolymer  -- I've updated the PR by removing changes from `config.yml`